go/service/incoming-share.go

```diff
@@ -42,19 +43,27 @@ func strPtr(str string) *string {
 	return nil
 }
 
-func (h *IncomingShareHandler) GetIncomingShareItems(_ context.Context) (items []keybase1.IncomingShareItem, err error) {
+func (h *IncomingShareHandler) GetIncomingShareItems(ctx context.Context) (items []keybase1.IncomingShareItem, err error) {
+	mctx := libkb.NewMetaContext(ctx, h.G())
 	manifestPath := filepath.Join(h.G().Env.GetMobileSharedHome(), "Library", "Caches", "incoming-shares", "manifest.json")
 	f, err := os.Open(manifestPath)
 	if err != nil {
+		mctx.Error("incoming-share: open manifest.json error: %v", err)
```

Comment:
Why not Debug ? (Throughout)

Reply:
I incorrectly thought Debug wasn't logged on mobile. Will update; thanks!